release_notes.md

```diff
@@ -1,6 +1,28 @@
-# Release Notes
+# Release Note 0.2.0
```

Comment:
Q. If this document will be 'release_notes.md', shouldn't we leave thie `# Release Notes` header and collect notes under this..?

Reply:
um... I heard this `release_note` will used on page each release page, like [this](https://github.com/Samsung/ONE-vscode/releases/tag/0.1.0). so, we just type version on main header, I thought.

Reply:
Oh... I thought it will be collected here, based on the document name.

Reply:
I think it's better to change the name in that case. @hyunsik-yoon I don't know the history of this file, so I'm just asking..